ddtrace/contrib/flask/patch.py

```diff
@@ -102,14 +102,20 @@ def patch():
         "handle_exception",
         "handle_http_exception",
         "handle_user_exception",
-        "try_trigger_before_first_request_functions",
         "do_teardown_request",
         "do_teardown_appcontext",
         "send_static_file",
     ]
```

Comment:
This approach better aligns with how we patch different versions of flask

```suggestion
    ]

    if flask_version < (2, 2, 0):
        flask_app_traces.append("try_trigger_before_first_request_functions")
```